src/test/java/bio/terra/service/snapshot/SnapshotConnectedTest.java

```diff
@@ -174,6 +175,10 @@ public void testEnumeration() throws Exception {
       snapshotList.add(summaryModel);
     }
 
+    // Reverse the order of the array since the order we return the snapshots in by default is
+    // descending order of creation
+    Collections.reverse(snapshotList);
```

Comment:
huh...I thought I replied earlier.  sorry!  It's because PG returns records based on where they physically live which happens to be in order of insertion.  It's not guaranteed however.  So really, it was luck that the tests passed.